source/extensions/filters/http/well_known_names.h

```diff
@@ -54,6 +54,8 @@ class HttpFilterNameValues {
   const std::string HeaderToMetadata = "envoy.filters.http.header_to_metadata";
   // Tap filter
   const std::string Tap = "envoy.filters.http.tap";
+  // Adaptive concurrency limit filter.
```

Comment:
_hyper_ nit: The other comments in this don't end with a `.`, maybe remove to match style?